shared/params/config.go

```diff
@@ -108,14 +108,14 @@ type DepositContractConfig struct {
 var defaultBeaconConfig = &BeaconChainConfig{
 	// Constants (Non-configurable)
 	FarFutureEpoch:           1<<64 - 1,
-	BaseRewardsPerEpoch:      5,
+	BaseRewardsPerEpoch:      4,
 	DepositContractTreeDepth: 32,
 	SecondsPerDay:            86400,
 
 	// Misc constant.
-	ShardCount:                     1024,
 	TargetCommitteeSize:            128,
-	MaxValidatorsPerCommittee:      4096,
+	MaxValidatorsPerCommittee:      2048,
```

Comment:
Why is this different? This isn't mentioned in your description.

Reply:
Updated descriptions